src/utils/HTMLUtils/index.js

```diff
@@ -52,10 +52,24 @@ const parseContent = html => {
   return { content };
 };
 
+const parseContents = html => {
+  const contents = cheerio(html)
+    .filter((i, elem) => getInnerHTML(elem) !== "")
+    .map((i, elem) => mapElementToObject(elem))
+    .toArray();
+
+  if (contents.length === 0) {
+    return;
+  }
+
+  return { contents };
+};
+
```

Comment:
Could this be merged into the other function somehow? Seems a bit weird to have two functions that do essentially the same thing.